cluster-autoscaler/simulator/predicatechecker/schedulerbased.go

```diff
@@ -184,6 +198,16 @@ func (p *SchedulerBasedPredicateChecker) CheckPredicates(clusterSnapshot cluster
 	return nil
 }
 
+func (p *SchedulerBasedPredicateChecker) frameworkForPod(pod *apiv1.Pod) schedulerframework.Framework {
+	fwk, ok := p.profiles[pod.Spec.SchedulerName]
+	if !ok {
+		klog.Warningf("profile not found for scheduler name %q", pod.Spec.SchedulerName)
```

Comment:
Avoid the use of `Warning`

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions

Reply:
use `InfoS`